src/cli/cli.cpp

```diff
@@ -3767,6 +3769,34 @@ otError Interpreter::ProcessTxPower(uint8_t aArgsLength, char *aArgs[])
     return error;
 }
 
+otError Interpreter::ProcessUptime(uint8_t aArgsLength, char *aArgs[])
+{
+    OT_UNUSED_VARIABLE(aArgs);
+
+    otError  error = OT_ERROR_NONE;
+    uint32_t now;
+    uint32_t seconds;
+    uint32_t minutes;
+    uint32_t hours;
+    uint32_t days;
+
+    VerifyOrExit(aArgsLength == 0, error = OT_ERROR_INVALID_ARGS);
+
+    now = static_cast<uint32_t>((otPlatTimeGet() - mStartTimeUs) / 1000000);
+
+    seconds = now % 60;
+    now     = now / 60;
+    minutes = now % 60;
+    now     = now / 60;
+    hours   = now % 24;
+    days    = now / 24;
+
+    OutputLine("%02d:%02d:%02d up %d days", hours, minutes, seconds, days);
```

Comment:
```suggestion
    OutputLine("%dd%dh%dm%ds", days, hours, minutes, seconds);
```
nit: formats like `3d20h30m59s` is easier to parse?